mapdata/WarcraftLegacies/AbilityData/Skin/842223700.json

```diff
@@ -0,0 +1,45 @@
+{
+  "OldId": 1718764353,
+  "NewId": 842223700,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1953653089,
+      "Type": 3,
+      "Value": "ReplaceableTextures/CommandButtons/BTNFlameStorm.blp"
+    },
+    {
+      "Id": 2020631137,
+      "Value": 1
+    },
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(Eredar Warlock)"
+    },
+    {
+      "Id": 2037082209,
+      "Type": 3,
+      "Value": "Q"
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Calls down 15 individual firebolts from the sky igniting units, affected units take 15 damage per second for 3 seconds."
```

Comment:
Switch this over to using raw data fields. This tutorial has a decent explanation if you're not familiar: https://www.hiveworkshop.com/threads/raw-data-tutorial.32324/ Otherwise you can ask another dev.
Funnily enough the burn actually lasts 2 seconds, not 3, which highlights the value of using raw data!

Reply:
Will do , I endeavour to most of the time but I seem to run into issues with the Extended Tooltips and the Learn Tooltips where the data just displays as 0. I'll have a browse of that tutorial, I imagine its glaring me in the face what I keep doing wrong!